tee-worker/enclave-runtime/src/initialization/mod.rs

```diff
@@ -295,7 +295,17 @@ fn run_vc_issuance() -> Result<(), Error> {
 	);
 	let extrinsic_factory = get_extrinsic_factory_from_integritee_solo_or_parachain()?;
 	let node_metadata_repo = get_node_metadata_repository_from_integritee_solo_or_parachain()?;
-	run_vc_handler_runner(Arc::new(stf_task_context), extrinsic_factory, node_metadata_repo);
+
+	let vc_handler = std::thread::spawn(move || {
```

Comment:
why do we need to spawn a new thread? `run_vc_issuance` also is called within an already spawned thread